wireapp/wire-android
# feat: show legal hold disabled dialog [WPB-4870]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-4870" title="WPB-4870" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />WPB-4870</a>  [Android] Disabling legal hold for a team member 
  </td></table>
  <br />
 














----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

Implemented the view model with logic to show proper dialog informing the user about legal hold being disabled for him.

### Dependencies (Optional)

Needs releases with:

- https://github.com/wireapp/kalium/pull/2299

### Testing

#### Test Coverage (Optional)


#### How to Test

Disable legal hold for the currently logged in user and then this dialog should pop up.

### Attachments (Optional)

https://github.com/wireapp/wire-android/assets/30429749/f1e12e08-3355-4b94-af2d-f11bbc43ac8a

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
